crates/sui-graphql-rpc/src/types/owner.rs

```diff
@@ -69,14 +70,16 @@ pub(crate) enum Owner {
     Object(Object),
 }
 
-pub(crate) struct AmbiguousOwner;
+pub(crate) struct AmbiguousOwner {
+    pub address: SuiAddress,
+}
 
 #[allow(unreachable_code)]
 #[allow(unused_variables)]
 #[Object]
 impl AmbiguousOwner {
-    pub async fn location(&self) -> SuiAddress {
-        unimplemented!()
+    pub async fn location(&self, ctx: &Context<'_>) -> SuiAddress {
+        self.address.clone()
```

Comment:
For our collective sanity, I do think it's worth just implementing `Copy` for our `SuiAddress` as well.